BunnymodXT/modules/ServerDLL.cpp

```
@@ -1825,6 +1836,23 @@ HOOK_DEF_4(ServerDLL, void, __fastcall, CPushable__Move, void*, thisptr, int, ed
 	ORIG_CPushable__Move(thisptr, edx, pOther, push);
 }
 
+void ServerDLL::DoWouldCrashMessage()
+{
+	// We send a user message from the ServerDLL so that we have the notice in the demo.
+	int gmsgTextMsg = pEngfuncs->pfnRegUserMsg( "TextMsg", -1 );
```

I'm assuming this is one of the standard usermsgs, does registering it return the same int or something? Is it ok to register every time?

---

Replaced with a simple server cmd "say" so that it's portable and just silently fails if "say" isn't a command in some mod.